hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/properties/MirrorNodeEvmProperties.java

```diff
@@ -58,6 +58,8 @@ public class MirrorNodeEvmProperties implements EvmProperties {
 
     private boolean dynamicEvmVersion = true;
 
+    private long exchangeRateGasReq = 100;
```

Comment:
Also add to docs/configuration.md.

```suggestion
    @Min(1)
    private long exchangeRateGasReq = 100;
```